config/routes/admin.rb

```diff
@@ -3,6 +3,8 @@
 
     authenticated :spree_user, -> user { user.admin? } do
       mount DelayedJobWeb, at: '/delayed_job'
+      # Mount feature toggle with flipper route
```

Comment:
I don't think this comment is necessary. It is in their README but I think it's quite clear here in this context what it does.